extensions/vault/runtime/src/main/java/io/quarkus/vault/VaultTransitSecretEngine.java

```diff
@@ -259,7 +259,7 @@
      * @return key detail, or null if the key does not exist
      * @see <a href="https://www.vaultproject.io/api-docs/secret/transit#read-key">read key</a>
      */
-    VaultTransitKeyDetail readKey(String keyName);
+    VaultTransitKeyDetail<?> readKey(String keyName);
```

Comment:
? extends VaultTransitKeyVersion

Reply:
you are correct. my bad. I did not think the compiler would accept:
```
        VaultTransitKeyDetail<?> vaultTransitKeyDetail = transit.readKey(KEY_NAME);
        VaultTransitKeyVersion version = vaultTransitKeyDetail.getVersions().get("1");
```
let's leave it as is.